cache/hhvm-scalable-cache.h

```diff
@@ -746,6 +756,19 @@ FastHashMonitor(){
         best_avg, best_size, tFC_hit_, best_option_tMiss);
     }
   }
+
+  if(best_option_tMiss < previousMissRatio){
+    exponent = 0;
```

Comment:
`exponentCap` right

Reply:
Interesting that this wasn't caught by the compiler, Dhruv and I did some digging and it appears the compiler is making optimizations under the hood, so just a heads up that it may not catch everything